cromwell-helm/values.yaml

```diff
@@ -46,5 +46,73 @@ db:
   user: postgres_user
   password: "SUPPLIED BY LEO"
 
+jobmanager:
+  name: jobmanager
+  applicationVersion: latest
+  ui:
+    image:
+      repository: us.gcr.io/broad-dsp-gcr-public/job-manager-ui
+      tag:
+    service:
+      type: ClusterIP
+      port: 8000
+    probes:
+      readiness:
+        # ui.probes.readiness.enable -- Whether to configure a readiness probe
+        enabled: true
+        # ui.probes.readiness.spec -- Kubernetes spec for readiness probe
+        spec:
+          httpGet:
+            path: /health
+            port: 8000
+          timeoutSeconds: 5
+          periodSeconds: 10
+          failureThreshold: 6
+          successThreshold: 1
+      liveness:
+        # ui.probes.liveness.enable -- Whether to configure a liveness probe
+        enabled: true
+        # ui.probes.liveness.spec -- Kubernetes spec for liveness probe
+        spec:
+          httpGet:
+            path: /health
+            port: 8000
+          timeoutSeconds: 5
+          periodSeconds: 10
+          failureThreshold: 30
+          successThreshold: 1
+  api:
+    image:
+      repository: us.gcr.io/broad-dsp-gcr-public/job-manager-api-cromwell
+      tag:
+    service:
+      type: ClusterIP
+      port: 8190
+    probes:
+      readiness:
+        # ui.probes.readiness.enable -- Whether to configure a readiness probe
+        enabled: false # TODO: enable after connecting to Cromwell -breilly 20220420
+        # ui.probes.readiness.spec -- Kubernetes spec for readiness probe
+        spec:
+          httpGet:
+            path: /api/v1/health
```

Comment:
I didn't think to connect to the running instance. I'll have to learn how to do that.

Reply:
The logs showed successful responses from at least some of the health checks. 🤷‍♂️  I'm going to add a TODO here to double-check these since, as you pointed out, it might make more sense to continue using the proxy which could impact how the health checks are handled, I guess.

Reply:
Oh, actually the comment above these lines was wrong. These are the API checks which are not currently enabled because they were failing. I assumed it was because JobManager couldn't connect to Cromwell, but maybe there are other problems as well.